src/components/Clusters/components/AddClusterWizard.js

```diff
@@ -95,6 +95,11 @@ export function AddClusterWizard({ kubeconfig, setKubeconfig, config }) {
 
   const onComplete = () => {
     try {
+      kubeconfig?.users.forEach(user => {
+        if (!user?.user?.exec?.args?.includes('--oidc-extra-scope=openid')) {
+          user?.user?.exec?.args?.push('--oidc-extra-scope=openid');
+        }
+      });
```

Comment:
Is there a specific reason why you don't want to go this way?

https://github.com/kyma-project/busola/pull/3289/files

Busola is trying to mimic the behavior of the `kubelogin` command, which does not require `openid` to be passed as `--oidc-extra-scope`. Imho you should implement it the same way.

Reply:
Hi Christian, thanks for your review and contribution.
Yes you are correct, I was unaware of your PR, next time you could attach it to the issue to make it more apparent :D
I have a follow-up question to you: should all the defined `--oidc-extra-scope` be passed along with the openid scope in the `scope` argument of the `UserManager` constructor?

Reply:
meaning if you have defined `--oidc-extra-scope=scope_a`, `--oidc-extra-scope=scope_a` should the scope passed to the oidc UserManager be `scope: "openid scope_a scope_a"`?

Reply:
Ok, I thought it would be enough to link the fix to the issue. Next time I'll put it right in the beginning of the issue description. You're right, the scopes have to be passed like this.

Reply:
thanks, I will incorporate your changes in my PR 👍